### DISCUSSION TITLE
osu!standard mod to prevent unnecessary/extra clicks when hitting circles

### DISCUSSION THREAD

**Discussion by StayBlue** [ppy/osu#17893] [Ideas]
I have checked to see if this is a duplicate and could not find an issue/discussion similar to this one.

Essentially, the mod would make it so you can't click more times than there are circles on the screen. To put it more simply, it would prevent people from clicking more times than is necessary.

While the idea I have in mind is not perfect, the player can be punished by counting extra clicks as misses and potentially giving a toggleable option for an auditory sound when a miss is caused by an extra click.

**Comment by peppy** [comment]
So basically miss when you click no circle?

**Reply by StayBlue** [reply]
Miss when you click extra times than necessary. If there is a triple and you click four times instead of three, for example, you'd get one miss.

**Reply by StayBlue** [reply]
You could still have technically hit all three circles, but the extra click would be considered redundant and would count as a miss.
